packages/block-library/src/blocks/supernova/scss/_block-alignments.scss

```diff
@@ -1,3 +1,55 @@
+/*
+ * Supernova is a spacial block, so
+ * it's alignment need some adjustments.
+ */
+
+.supernova {
+  
+  /*
+   * Supernova has always alignfull applied on it
+   * so it's children need to have
+   * some padding left and right.
+   */
+  > * {
+    padding-left: var(--nb-container-spacings);
+    padding-right: var(--nb-container-spacings);
+  }
+  
+  /*
+   * .alignfull {
+   *    margin-left: calc(-1 * var(--nb-container-spacings))
+   *    margin-right: calc(-1 * var(--nb-container-spacings))
+   * }
+   *
+   * A nested alignfull class nested inside another alignfull
+   * will cause some trouble with it's negative margin,
+   * so we choose to reset it.
+   */
+  > .alignfull {
```

Comment:
This would probably not be needed is the pattern of `__inner-container > .wp-block` was followed since there would be no elements with negative margins without a parent with sides padding